resources/operator_store_results.yaml

```diff
@@ -50,6 +52,3 @@ spec:
           volumeMounts:
             - mountPath: /redis-master-data
               name: data
-      volumes:
```

Comment:
This was for the redis volume? You think they should share?

Reply:
The containers aren't sharing them, but all the volumes need to be defined together and was throwing an error and this volume is mounted only for redis container.